contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -435,6 +440,44 @@
       clearSelections() {
         this.selected = [];
       },
+
+      sortNodes(selected) {
+        const selectedNodes = selected.map(id => this.getContentNode(id));
+        const orderedNodes = selectedNodes.sort(this.compareNodeTitles);
+
+        const nodeX = this.findNodeBeforeFirstSelected(orderedNodes);
+
+        const targetParent = this.node.id;
+        const targetPosition = nodeX
+          ? RELATIVE_TREE_POSITIONS.RIGHT
+          : RELATIVE_TREE_POSITIONS.FIRST_CHILD;
+
+        orderedNodes.forEach(node => {
+          const position =
+            targetPosition === RELATIVE_TREE_POSITIONS.RIGHT
+              ? targetPosition
+              : RELATIVE_TREE_POSITIONS.LAST_CHILD;
+          this.moveContentNodes({
+            id__in: [String(node.id)],
+            parent: targetParent,
```

Comment:
First, `moveContentNodes` doesn't accept a `parent` argument. It should be `target` instead. 

Second, when `position` is `FIRST_CHILD`, it should be the parent as you have here, but if the position is `RIGHT`, the target should be `nodeX`. That's because the `RIGHT` position is relative to another element, i.e. as a sibling.